src/commands/viewProperties.ts

```diff
@@ -14,7 +15,10 @@ import { SlotTreeItem } from '../tree/SlotTreeItem';
 
 export async function viewProperties(context: IActionContext, node?: SlotTreeItem | RemoteFunctionTreeItem | LocalFunctionTreeItem): Promise<void> {
     if (!node) {
-        node = await ext.rgApi.tree.showTreeItemPicker<SlotTreeItem>(new RegExp(ResolvedFunctionAppResource.productionContextValue), context);
+        node = await ext.rgApi.pickAppResource<SlotTreeItem>(context, {
```

Comment:
We're getting rid of this command in favor of RG, right?

Reply:
Yeah I was going to do that afterwards

Reply:
So actually, the view properties command needs to still exist, but just not on the app resource. So it should never show up in the tree.